apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -12,49 +12,40 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <% if Configuration.turbolinks_enabled? %>
-    <meta name="turbolinks-root" content="<%= ENV['RAILS_RELATIVE_URL_ROOT'] || "/" %>">
+  <meta name="turbolinks-root" content="<%= ENV['RAILS_RELATIVE_URL_ROOT'] || "/" %>">
   <% end %>
-<%= render partial: '/layouts/nav/styles', locals: { bg_color: Configuration.brand_bg_color, link_active_color: Configuration.brand_link_active_bg_color } %>
+  <%= render partial: '/layouts/nav/styles', locals: { bg_color: Configuration.brand_bg_color, link_active_color: Configuration.brand_link_active_bg_color } %>
 </head>
 <body>
   <header <%= 'data-turbolinks-permanent' if Configuration.turbolinks_enabled? %>>
-    <nav class="navbar navbar-<%= ENV['OOD_NAVBAR_TYPE'].present? ? ENV['OOD_NAVBAR_TYPE'] : 'inverse' %> navbar-static-top">
-    <div class="container-fluid">
-      <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-9" aria-expanded="false"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
-        <% if ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'].present? %>
-          <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img src="<%= ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'] %>"></a>
-        <% else %>
-          <a class="navbar-brand" href="<%= root_path %>"><%= OodAppkit.dashboard.title.html_safe %></a>
-        <% end %>
-      </div>
-      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-9">
-        <ul class="nav navbar-nav">
+    <nav class="navbar navbar-expand-md navbar-dark navbar-color">
```

Comment:
This is always `navbar-dark`. Previously we supported using `OOD_NAVBAR_TYPE` as "default" or "inverse" to specify `navbar-default` or `navbar-inverse`. Is the bootstrap 4 version of `navbar-default` the class `navbar-light` and `navbar-inverse` the class `navbar-dark`?

Reply:
Different branding needs require a light or a dark background.

Reply:
Yeah the `navbar-dark` naming convention that Bootstrap 4 uses isn't my favorite. https://getbootstrap.com/docs/4.5/components/navbar/#color-schemes

`navbar-dark` in this context changes the color of `<a>` tags, not the background. I'll make sure that we remain API compatible and `OOD_NAVBAR_TYPE` will stay relevant